config/locales/bn.yml

```diff
@@ -23,7 +23,7 @@ bn:
     hosted_on: এই মাস্টাডনটি আছে %{domain} এ
     instance_actor_flash: 'এই অ্যাকাউন্টটি ভার্চুয়াল এক্টর যা নিজে কোনও সার্ভারের প্রতিনিধিত্ব করতে ব্যবহৃত হয় এবং কোনও পৃথক ব্যবহারকারী নয়। এটি ফেডারেশনের উদ্দেশ্যে ব্যবহৃত হয় এবং আপনি যদি পুরো ইনস্ট্যান্স ব্লক করতে না চান তবে অবরুদ্ধ করা উচিত নয়, সেক্ষেত্রে আপনার ডোমেন ব্লক ব্যবহার করা উচিত।
 
-'
+      '
```

Comment:
why changes this line?

Reply:
Wow, that's why CI complained about normalization. I didn't see these changes, because apparently my Git client ignores whitespaces. That was added by normalization though, I ran it and then staged the changes. Will fix.